pages.go

```diff
@@ -178,6 +178,26 @@ func (f FullPath) DestinationAsset() AssetMinimal {
 	}
 }
 
+// SameAsset returns true if source and destination assets are the same
+// (and there are no intermediate steps, which should never happen, but
+// just in case something insane happens).
+func (f FullPath) SameAsset() bool {
+	if len(f.Path) > 0 {
+		return false
```

Comment:
This doesn't mesh with the comment about insanity

Reply:
Ah, ok, on another interpretation of the comment it does. But I don't know why you would want this. Recommend deleting this block
```
	if len(f.Path) > 0 {
		return false 
        }
```